gamemode/server/rounds.lua

```diff
@@ -89,21 +89,21 @@ local function InRound()
 	local hunters = GetLivingPlayers(TEAM_HUNTERS)
 	local props = GetLivingPlayers(TEAM_PROPS)
 
-	if( #props == 0 ) then
-		round.state = ROUND_END
-		round.endTime = CurTime()
-		round.winner = "Hunters"
-		hook.Call( "OBJHUNT_RoundEnd" )
-		return
-	end
-
-	if( #hunters == 0 ) then
-		round.state = ROUND_END
-		round.endTime = CurTime()
-		round.winner = "Props"
-		hook.Call( "OBJHUNT_RoundEnd" )
-		return
-	end
+	 if( #props == 0 ) then
+		 round.state = ROUND_END
+		 round.endTime = CurTime()
+		 round.winner = "Hunters"
+		 hook.Call( "OBJHUNT_RoundEnd" )
+		 return
+	 end
+
+	 if( #hunters == 0 ) then
+		 round.state = ROUND_END
+		 round.endTime = CurTime()
+		 round.winner = "Props"
+		 hook.Call( "OBJHUNT_RoundEnd" )
+		 return
+	 end
```

Comment:
That's odd, I am wondering if the carriage returns and line feeds were normalized when I was using visual studio.  Nevertheless, I will take care of this.